.rubocop_todo.yml

```diff
@@ -1,154 +1,106 @@
-require: rubocop-rspec
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2017-06-22 15:12:41 -0700 using RuboCop version 0.49.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
 
-Security/MarshalLoad:
-  Exclude:
-    - 'app/models/concerns/hyrax/user.rb'
-
-Metrics/ClassLength:
-  Exclude:
-    - 'app/actors/hyrax/actors/file_set_actor.rb'
-    - 'app/controllers/hyrax/file_sets_controller.rb'
-    - 'app/forms/hyrax/forms/permission_template_form.rb'
-    - 'app/presenters/hyrax/work_show_presenter.rb'
-    - 'app/services/hyrax/user_stat_importer.rb'
-    - 'lib/generators/hyrax/templates/catalog_controller.rb'
-    - 'lib/generators/hyrax/install_generator.rb'
-    - 'lib/hyrax/configuration.rb'
-
-Metrics/ParameterLists:
+# Offense count: 1
+Lint/ScriptPermission:
   Exclude:
-    - 'app/jobs/batch_create_job.rb'
+    - 'Rakefile'
 
+# Offense count: 104
 Metrics/AbcSize:
-  Exclude:
-    - 'app/indexers/hyrax/file_set_indexer.rb'
-    - 'app/services/hyrax/workflow/permission_query.rb'
+  Max: 58
```

Comment:
This is increasing from the limit we previously set of 28.  There were only 2 exceptions to that. I prefer we put that back so that we maintain a relatively low ceiling with a few outliers.